app/components/find/courses/about_schools_component/view.rb

```diff
@@ -38,6 +38,21 @@ def show_scitt_guidance?
           course_information_config.show_placement_guidance?(:program_type)
         end
 
+        def placements_url
+          if course.has_unpublished_changes? || (course.is_published? && course.is_running?)
+            URI.join(
+              Settings.search_ui.base_url,
```

Comment:
I think this way of setting the URL breaks review app when it comes to redirection to find. Should work fine in production I think. We use the same logic for `View on find` button

Reply:
How odd...